File: WWW/DesktopModules/NVCMS.TinTuc/Display/ViewPage/Main.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.IO;
using System.Text.RegularExpressions;
using DotNetNuke;
using DotNetNuke.Entities;
using DotNetNuke.Entities.Modules;
using DotNetNuke.Common;
using DotNetNuke.Entities.Portals;
using DotNetNuke.Common.Utilities;
using NVCMS.Modules.TinTuc;
namespace DesktopModules.TinTuc.ViewPage
{
    public partial class Main : PortalModuleBase
    {
        protected void Page_Load(object sender, EventArgs e)
        {
            string fbclid = Request.QueryString["fbclid"];
            string sUrl = Request.RawUrl.Replace("?fbclid=" + fbclid, "");
            int sId = Ultis.GetRequestId(sUrl);
            PortalModuleBase control = new PortalModuleBase();
            if (sId == -1) //Index
            {
                control = Page.LoadControl("/DesktopModules/NVCMS.TinTuc/Display/ViewPage/Index.ascx") as PortalModuleBase;
            }
            else  //Details
            {
                control = Page.LoadControl("~/DesktopModules/NVCMS.TinTuc/display/Detail/Detail.ascx") as PortalModuleBase;
            }
            //Response.Write(sId);
            control.ModuleConfiguration = ModuleConfiguration;
            plhNews.Controls.Add(control);
        }
    }
}

File: WWW/DesktopModules/NVCMS.Marketing/Manager/ZaloSend/Main.ascx.vb
Namespace NVCMS.Modules.Marketing
    Partial Class ZaloSendMain
        Inherits Entities.Modules.PortalModuleBase

        Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
            Try
                Dim dynamicPage As String = DotNetNuke.Common.ResolveUrl(Me.TemplateSourceDirectory & "/Viewer.ascx")
                Dim objModule As Entities.Modules.PortalModuleBase = CType(Me.LoadControl(dynamicPage), DotNetNuke.Entities.Modules.PortalModuleBase)
                If Not objModule Is Nothing Then
                    objModule.ModuleConfiguration = Me.ModuleConfiguration
                    phDynamicPlaceHolder.Controls.Add(objModule)
                End If
            Catch ex As Exception
                ProcessModuleLoadException(Me, ex)
            End Try
        End Sub

    End Class
End Namespace